scripts/lib.sh

```diff
@@ -32,6 +32,24 @@ function await_server_on_port() {
   done
 }
 
+# Check that core-lightning block-proccessing is caught up
+# CLI integration tests should call this before attempting to pay invoices
+function await_cln_block_processing() {
+  EXPECTED_BLOCK_HEIGHT="$($FM_BTC_CLIENT getblockchaininfo | jq -r '.blocks')"
+
+  # ln1
+  until [ $EXPECTED_BLOCK_HEIGHT == "$($FM_LN1 getinfo | jq -r '.blockheight')" ]
+  do
+      sleep $POLL_INTERVAL
+  done
```

Comment:
I feel like an abstraction over this construct would be useful, e.g. `await "$EXPECTED_BLOCK_HEIGHT == \"$($FM_LN1 getinfo | jq -r '.blockheight')\""`

Reply:
Like a separate bash function that can just await a condition?